dev/packages/alpha/system/0.0.1/manifest.yml

```diff
@@ -0,0 +1,48 @@
+format_version: 1.0.0
+name: system
+title: System
+version: 0.0.1
+license: basic
+description: System Integration
+type: integration
+categories:
+- logs
+- metrics
+release: experimental
```

Comment:
Make it directly beta, otherwise it will not show up by default. @kaiyan-sheng This might also apply for you.